chemprop/nn/transforms.py

```diff
@@ -0,0 +1,24 @@
+from sklearn.preprocessing import StandardScaler
+import torch
+from torch import nn, Tensor
+from numpy.typing import ArrayLike
+from lightning.pytorch.core.mixins import HyperparametersMixin
+
+from chemprop.nn.hparams import HasHParams
+
+class OutputTransform(nn.Module):
+    mean: Tensor
+    scale: Tensor
+
+    def __init__(self, mean: ArrayLike, scale: ArrayLike):
+        super().__init__()
+        
+        self.register_buffer('mean', torch.tensor(mean, dtype=torch.float32))
+        self.register_buffer('scale', torch.tensor(scale, dtype=torch.float32))
```

Comment:
```suggestion
        self.register_buffer('mean', torch.tensor(mean, dtype=torch.float))
        self.register_buffer('scale', torch.tensor(scale, dtype=torch.float))
```
does this work? Don't want to pin ourselves to a precision here--better to let `torch` decide what's best.